### DISCUSSION TITLE
Sei

### DISCUSSION THREAD

**Discussion by RobinBluehub** [pybamm-team/PyBaMM#2179] [Q&A] [answered]
How to set the single-layer sei film in pybamm, the DFN model defaults to double-layer, right?

**Comment by valentinsulzer** [comment, marked as answer]
Look for the parameters relating to the inner layer (e..g Initial inner SEI thickness [m], Inner SEI reaction proportion) and set those to zero. Then the inner SEI layer will always be zero and you just have the outer SEI layer.
